### PR TITLE
Fix null `is_invoice` issue

### PR DESCRIPTION
**Pull request by ashkan18**
# Problem
On https://github.com/artsy/metaphysics/pull/686 we added `is_invoice` to messages response, using destructing to check if `metadata` has `lewitt_invoice_id` caused returning `null` when messages don't have `lewitt_invoice_id`, we want to return `false` for `is_invoice` for these cases,

# Solution
First check if we don't have metadata return `false` and if we do then check for `isExisty`.

### CODE REVIEW

File: schema/me/conversation/message.js [hunk 1]
```diff
@@ -44,7 +44,10 @@ export const MessageType = new GraphQLObjectType({
     is_invoice: {
       description: "True if message is an invoice message",
       type: GraphQLBoolean,
-      resolve: ({ metadata: { lewitt_invoice_id } }) => isExisty(lewitt_invoice_id),
+      resolve: ({ metadata }) => {
+        if (!metadata) return false
+        return isExisty(metadata.lewitt_invoice_id)
+      },
```
Comment (alloy) [inline schema/me/conversation/message.js]: Could make this a bit more idiomatic with `!!metadata && isExisty(metadata.lewitt_invoice_id)`.

Comment (alloy) [inline schema/me/conversation/message.js]: Or even `!!metadata && !!metadata.lewitt_invoice_id` ?

**Comment by alloy** [comment]
Small code comment, but otherwise LGTM 👍 Feel free to merge whenever.

File: schema/me/conversation/message.js (hunk 1 above)
Comment (ashkan18) [inline schema/me/conversation/message.js]: `!!` always confuses me hehe, going with your first suggestion with one less `!!`

**Comment by ashkan18** [comment]
Merging 👍

**Pull request merged**